resources/navigating_WAI/scope.md

```diff
@@ -13,26 +13,38 @@ https://github.com/w3c-cg/aar/issues/6
 
 ## Scope
 
-TBD
+Creating new tags for existing WAI pages for users to search for or filter on.  OR create a new resource with links to specific resources or sections of the WAI website based on need and usability level.   
 
 ## Audience
 
-### Primary
+Anyone interested in learning about or more about web accessibility at any stage of their career. 
```

Comment:
Can we update this to "Anyone interested in learning about (or more about) web accessibility at any stage of their career."

Reply:
Yes